.github/workflows/composites/maven-build-with-dry-run-for-tests/action.yaml

```diff
@@ -4,13 +4,13 @@ runs:
   using: "composite"
   steps:
 
-    - name: run 'package' on the project
+    - name: run 'install' on the project
       shell: bash
       run: |
         ./mvnw install -B  \
               -Dskip.build.image=true \
               -DskipTests -DskipITs \
-              -T 1C -q
+              -T 1C -U -q
```

Comment:
Doesn't `-U` tell maven to update dependencies?

Reply:
Makes sense.  I will just point out there is the potential...due to bad timing that we could finish the build locally and install those snapshots but inbetween that time and this command we could publish new snapshots to repo.spring.io and pull down those as well (for k8s).  Although the chance of that is quite small and I am not sure there is anything we can really do about it.